src/borrow.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+
+let borrowrawdata = fs.readFileSync('./src/data/borrow.json');  
+let borrow = JSON.parse(borrowrawdata);
+let rawdata = fs.readFileSync('./src/data/books.json');  
+let books = JSON.parse(rawdata);  
+let userrawdata = fs.readFileSync('./src/data/users.json');  
+let users = JSON.parse(userrawdata);
+
+function getborrowList(userID){
```

Comment:
this method should return all the books that are rented. So u don't need to pass the user id.